src/lib.rs

```diff
@@ -578,15 +578,14 @@ To override the toolchain mounted in the image, set `target.{}.image.toolchain =
             // set the sysroot explicitly to the toolchain
             let mut is_nightly = toolchain.channel.contains("nightly");
 
-            let installed_toolchains = rustup::installed_toolchains(msg_info)?;
```

Comment:
This is removed entirely, which is needed further below.

Reply:
I reworked the code; it is not used now.